generated/nidaqmx/_task_modules/ai_channel_collection.py

```diff
@@ -1503,13 +1503,15 @@ def add_ai_rosette_strain_gage_chan(
                 measurement.
         Returns:
             nidaqmx._task_modules.channels.ai_channel.AIChannel:
-            
+
             Indicates the newly created channel object.
         """
         if rosette_meas_types is None:
             rosette_meas_types = []
 
-        rosette_meas_types = numpy.int32([p.value for p in rosette_meas_types])
+        temp_vals = [p.value for p in rosette_meas_types]
+
+        rosette_meas_types = numpy.int32(temp_vals)
```

Comment:
@bkeryan  @zhindes Help needed for the mypy check error. 

![image](https://github.com/ni/nidaqmx-python/assets/27721199/ce04a4e7-87f2-4f8a-90f2-ca2d9419e579)

I tried type hint `List[StrainGageRosetteMeasurementType]` to `rosette_meas_types`, but I still get the similar error. Any suggestions? 
![image](https://github.com/ni/nidaqmx-python/assets/27721199/d99f6238-ce2a-400d-984d-23be54f88a61)
![image](https://github.com/ni/nidaqmx-python/assets/27721199/c4c6f50f-9f54-4f1b-9349-3d3a1933ac40)

![image](https://github.com/ni/nidaqmx-python/assets/27721199/0ca87791-e685-4181-84d6-3205e2468a81)

Reply:
I think the solution is to type `temp_vals` like this:


```python
temp_vals: List[int] = [p.value for p in rosette_meas_types]
```

Reply:
Rerunning codegen on your branch does this:
```
-        temp_vals = [p.value for p in rosette_meas_types]
-
-        rosette_meas_types = numpy.int32(temp_vals)
+        rosette_meas_types = numpy.int32([p.value for p in rosette_meas_types])
```

Which tells me two things:
- I missed one of the codegen templates
- build.yml should check for dirty codegen *before* it runs linters :)

Reply:
You need to update this: https://github.com/ni/nidaqmx-python/blob/master/src/codegen/utilities/function_helpers.py#L133

If `param.ctypes_data_type.startswith("numpy.")` then it should generate `numpy.array({1}, dtype={0})`, and likewise for the array version.

Reply:
@bkeryan @zhindes Thanks for the suggestions.

For some reasons, I can't use `numpy.array({1}, dtype={0})` for `rosette_meas_types` (enum list). As a generated code `rosette_meas_types = numpy.array(rosette_meas_types, dtype=numpy.int32)` causing pytest `Type Error` failure. I have to extract value from enum list with `numpy.array([p.value for p in {0}], dtype={1})`.

![image](https://github.com/ni/nidaqmx-python/assets/27721199/ce34a3d3-fb0f-43cd-9cae-319d89df294a)

Reply:
That's right. I should have said "and likewise for the enum-array version"